pkg/apis/pipeline/v1alpha1/contexts.go

```diff
@@ -28,3 +28,25 @@ type hdcnKey struct{}
 func WithDefaultConfigurationName(ctx context.Context) context.Context {
 	return context.WithValue(ctx, hdcnKey{}, struct{}{})
 }
+
+// HasDefaultConfigurationName checks to see whether the given context has
+// been marked as having a default configurationName.
+func HasDefaultConfigurationName(ctx context.Context) bool {
+	return ctx.Value(hdcnKey{}) != nil
```

Comment:
what's `hdcn` ?

Reply:
why not ? :joy_cat: `hdcn` is for `hasdefaultconfigurationname`

Reply:
🤭